docker-compose.yml

```diff
@@ -122,7 +122,8 @@ x-hierarchy:
   - postgres
   - python-wheel-manylinux-2010
   - python-wheel-manylinux-2014
-  - python-wheel-manylinux-test
+  - python-wheel-manylinux-test-imports
```

Comment:
I had to separate the two testing steps due to the lack of pandas wheels for aarch64. Compiling pandas takes significant time, so I rather chose to cache it in a docker image. 

Installing the pyarrow wheels in an environment with preinstalled dependencies like numpy wouldn't test the setup.py config, so IMO it's better to isolate these to two separate cases.